apps/chat/src/components/Common/ConfirmDialog.tsx

```diff
@@ -36,17 +38,29 @@ export const ConfirmDialog = ({
       dismissProps={{ outsidePressEvent: 'mousedown' }}
       hideClose
     >
-      <div className="flex flex-col gap-2">
+      <div className="flex w-full flex-col gap-2 text-start">
         <h2 id={headingId} className="text-base font-semibold">
           {heading}
         </h2>
-        {description && (
-          <p id={descriptionId} data-qa="confirm-message">
-            {description}
-          </p>
-        )}
+        <div>
+          {description && (
+            <p
+              id={descriptionId}
+              data-qa="confirm-message"
+              className="text-secondary"
+            >
+              {description}
+            </p>
+          )}
+          {!!subDescriptions &&
+            subDescriptions.map((subDescription) => (
+              <p key={subDescription} className="text-secondary">
+                {subDescription}
+              </p>
+            ))}
+        </div>
```

Comment:
we can use line breaks with `whitespace-pre-line` instead of `subDescriptions` (+ `leading-3` for line-height)
```suggestion
            <p
              id={descriptionId}
              data-qa="confirm-message"
              className="text-secondary leading-3 whitespace-pre-line"
            >
              {description}
            </p>
          )}
        </div>
```